cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/generic/DynamicModel.java

```diff
@@ -49,6 +56,17 @@ public <T> T getParameter(String key, Class<T> clazz) {
         return clazz.cast(parameters.get(key));
     }
 
+    public <T> T getParameterWithFallback(String key, Class<T> clazz) {
+        try {
+            return clazz.cast(parameters.get(key));
```

Comment:
I would replace this with calling `getParameter` to avoid duplication

Reply:
very good catch, thank you @Bajzathd !